api/band/oracle/v1/query_grpc.pb.go

```diff
@@ -62,7 +62,7 @@ type QueryClient interface {
 	Reporters(ctx context.Context, in *QueryReportersRequest, opts ...grpc.CallOption) (*QueryReportersResponse, error)
 	// ActiveValidators queries all active oracle validators.
 	ActiveValidators(ctx context.Context, in *QueryActiveValidatorsRequest, opts ...grpc.CallOption) (*QueryActiveValidatorsResponse, error)
-	// Params queries parameters used for runnning bandchain network.
+	// Params queries parameters used for running bandchain network.
```

Comment:
please fix it in proto/band/oracle/v1/query.proto and generate this file using command `make proto-gen`